snake_env.py

```diff
@@ -49,7 +49,8 @@ def step(self, action):
         reward = 0
         
         observation = self._get_obs()
-        # TODO: add reward for moving towards apple
+        
+        reward = 1 / self.state.dist_to_apple
```

Comment:
Divide by zero error inc!
Put this in an `else` clause of the `ate` check :100: